proposals/interface-types/working-notes/instr.md

```diff
@@ -0,0 +1,87 @@
+# Interface Type Instructions
+
+## Lifting and lowering
+
+### ixx-to-sxx
+
+The numeric lifting and lowering instructions map between WASM's view of numbers and IT's view of numbers.
+
+
+| | | |
+| ----- | ----------- | ---------- |
+| `i32-to-s8` | `i32` .. => `s8` .. | Lift least significant 8 bits as signed 8 bit integer |
+| `i32-to-s8x` | `i32` .. => `s8` .. | Lift least 8 bits as signed 8 bit integer, error if more than 7 bits significant  |
```

Comment:
I don't recall discussing this `x`-suffixed variant before.  I'm not necessarily opposed, or anything, but, I'm still a little confused; it seems like the goal is to trap instead of silently ignoring bits.  Two thoughts/questions:
* shouldn't there by `x`-suffixed variants for both `s8` and `u8`?  (I only see the former)
* `x` as a suffix seems vaguely suggestive of "extend", maybe `t` for "trapping"?